filebeat/fileset/modules.go

```diff
@@ -168,7 +173,7 @@ func mcfgFromConfig(cfg *common.Config) (*ModuleConfig, error) {
 
 	mcfg.Filesets = map[string]*FilesetConfig{}
 	for name, filesetConfig := range dict {
-		if name == "module" || name == "enabled" {
+		if name == "module" || name == "enabled" || name == "path" {
```

Comment:
Why is this change needed?

Reply:
I order to allow ufcg to resolve the `$path.config` variables they must be part of the configuration. So after merging them in you end up with something like:

```
  module: xyz
  enabled: true
  dataset1:
    var.something: ${path.config}/some/resource.csv
  dataset2:
    enabled: true
    var.foo: bar
  path:
    home: /usr/share/filebeat
    config: /etc/filebeat
    data: y
    logs: z
```

Reply:
Oh I see :+1: